compatibility/compatibility-tests/build.gradle.kts

```diff
@@ -55,6 +55,6 @@ tasks.withType<Test>().configureEach {
 // with uncaught exception of type std::__1::system_error: mutex lock failed: Invalid argument`
 //
 // Compatibility tests fail, because Windows not supported by testcontainers (logged message)
-if (Os.isFamily(Os.FAMILY_MAC) || Os.isFamily(Os.FAMILY_WINDOWS)) {
+if ((Os.isFamily(Os.FAMILY_MAC) || Os.isFamily(Os.FAMILY_WINDOWS)) && System.getenv("CI") != null) {
```

Comment:
Why this one effectively disabling CI for this project?

Reply:
This is actually _enabling_ the tests to run on my Mac. CI was already disabled previously.

(and btw the tests run just fine on Mac, I'm not sure about the comment line 55).